spec/_attachments/ic.did

```diff
@@ -242,4 +259,8 @@ service ic : {
   }) -> (record {canister_id : canister_id});
   provisional_top_up_canister :
     (record { canister_id: canister_id; amount: nat }) -> ();
+
+  // canister loging
+  fetch_logs : (record { canister_id }) -> (fetch_logs_response);
+  fetch_logs_query : (record { canister_id }) -> (fetch_logs_response) query;
```

Comment:
Why do we need separate endpoints for the query and non-query? It's sufficient to have the single query endpoint, since all queries can be called in replicated mode.

Reply:
I added two endpoints looking at the bitcoin endpoints as an example:

```
  bitcoin_get_balance: (get_balance_request) -> (satoshi);
  bitcoin_get_balance_query: (get_balance_request) -> (satoshi) query;
  bitcoin_get_utxos: (get_utxos_request) -> (get_utxos_response);
  bitcoin_get_utxos_query: (get_utxos_request) -> (get_utxos_response) query;
```